### ISSUE TITLE
Next episode does not play next episode

### ISSUE THREAD

**Issue by neingeist** [fredrikburmester/streamyfin#172]
**Describe the bug**
In 0.17.0, when an episode ends, the player attempts to play the next episode. But it's not actually the next episode that's playing but the same that finished. Only the title text is updated to the next episode. 

**To Reproduce**
1. Play episode to the end
2. Pay attention which episode starts playing then

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy Tab A8 
 - OS: Android 14
 - Version 0.17.0 (GitHub release APK)

**Comment by fredrikburmester** [comment]
Thank you for this! Will fix.

**Comment by neingeist** [comment]
Sometimes, there isn't even any next episode starting - can't tell under what circumstances.

**Comment by fredrikburmester** [comment]
ok thank you guys, will look into it

**Comment by jd1** [comment]
Thanks for this bug, I was really confused when this happened to me and at first thought I had the same episode twice, but with a different title.

**Comment by jd1** [comment]
This is still an issue.

**Comment by fredrikburmester** [comment]
Can you try on the latest version?

**Comment by bwees** [comment]
I still get a infinite loading screen on next episode

**Comment by mathieures** [comment]
Letting next episode play or clicking on the button mostly gives me a black screen without any sound on version 0.26.1 (or according to the app setting, N/A).

**Comment by neingeist** [comment]
Can't test it currently, as 0.28.0 is somewhat broken for me (interface is only half the screen, no controls in the player).

**Comment by bwees** [comment]
still an issue

**Comment by Simon-Eklundh** [comment]
That's weird... It is working for me....

Is it a "real show" with only episodes without decimals in the episode numbers?

**Comment by bwees** [comment]
I thought it was next episode that wasn't working, it turns out it's actually the behavior of "skip credits". I would expect it to skip to the end and autoplay to the next episode but it doesn't do anything

**Comment by Simon-Eklundh** [comment]
Ah

I'd need to check that but last I tested I think it worked....

**Comment by bwees** [comment]
> Ah
> 
> I'd need to check that but last I tested I think it worked....

I just tested it across 4 shows in my library and they all show a skip credits button, when I tap it the player pauses but does not play the next episode.

**Comment by Simon-Eklundh** [comment]
Tested it on one of my shows just now, latest release (0.33) on android 

Skip credits button showed up, I clicked it, it skipped forward to the end of the credits, and then I had an after-credit scene which I could also skip using the "next episode" button

**Comment by bwees** [comment]
Yeah skip credits appears to not work if there is no end credits scene (ie end of the "credits" scene is the end of the episode)
